test/functional/feature_llmq_rotation.py

```diff
@@ -231,6 +231,23 @@ def run_test(self):
         self.wait_until(lambda: self.nodes[0].getbestblockhash() == new_quorum_blockhash)
         assert_equal(self.nodes[0].quorum("list", llmq_type), new_quorum_list)
 
+        self.log.info("Test 'quorum rotationinfo' RPC")
+        genesis_blockhash = self.nodes[0].getblockhash(0)
+        block_count = self.nodes[0].getblockcount()
+        hmc_base_blockhash = self.nodes[0].getblockhash(block_count - (block_count % 24) - 24 - 8)
+        best_block_hash = self.nodes[0].getbestblockhash()
+        rpc_qr_info = self.nodes[0].quorum("rotationinfo", best_block_hash, False, hmc_base_blockhash)
+        self.log.info(rpc_qr_info)
```

Comment:
I think there's no use to log it as it is; it looks like a leftover  from writing & debugging.

```diff
-self.log.info(rpc_qr_info)
```